README.md

```diff
@@ -45,6 +45,31 @@ about them:
 Similarly, you may need to `npm link path/to/your/js/sdk` in your `matrix-react-sdk`
 directory.
 
+Example of installation and link for vector:
+
+```
+mkdir ~/matrix
+cd ~/matrix
+git clone https://github.com/vector-im/vector-web.git
+cd ~/vector-web
+git checkout develop
```

Comment:
We do specify the github develop dependency of react/js sdk in package.json when we require something that hasn't been released yet, so you probably fetched develop js sdk without noticing. Yeah, cloning and building js sdk, then react sdk, then npm link probably makes sense. Don't forget to npm link js-sdk to react-sdk too, and don't forget to npm i source-map-loader in react-sdk and js-sdk.